src/game/g_target.cpp

```diff
@@ -1568,43 +1569,42 @@ void target_remove_portals_use(gentity_t *self, gentity_t *other,
     return;
   }
 
-  auto found = false;
+  auto hadActivePortals = false;
 
   if (activator->portalBlue) {
     G_FreeEntity(activator->portalBlue);
     activator->portalBlue = nullptr;
-    found = true;
+    hadActivePortals = true;
   }
 
   if (activator->portalRed) {
     G_FreeEntity(activator->portalRed);
     activator->portalRed = nullptr;
-    found = true;
+    hadActivePortals = true;
   }
 
-  if (!found) {
-    return;
-  }
+  if (hadActivePortals) {
+    if (self->spawnflags & SF_REMOVE_PORTALS_ACTIVATE_TARGETS) {
+      G_UseTargets(self, activator);
+    }
 
-  // activate targets if any portals were reset
-  gentity_t *ent = G_PickTarget(self->target);
-  if (ent && ent->use) {
-    G_UseEntity(ent, self, activator);
-  }
+    // play sound to client
+    if (self->noise_index) {
+      gentity_t *noiseEnt = ETJump::soundEvent(
+          activator->r.currentOrigin,
+          EV_GENERAL_CLIENT_SOUND_VOLUME,
+          self->noise_index);
 
-  // play sound to client
-  if (self->noise_index) {
-    gentity_t *noiseEnt = ETJump::soundEvent(
-        activator->r.currentOrigin,
-        EV_GENERAL_CLIENT_SOUND_VOLUME,
-        self->noise_index);
+      noiseEnt->s.onFireStart = 255;
```

Comment:
This should probably have a `volume` key associated with it? Since you're sending a volume controlled event anyway.